f/chunks.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function chunks(arr, size) {
+  size = size >> 0;
+  if ( size <= 0 || size > arr.length ) return arr;
+  const buffer = new Array(Math.ceil(arr.length / size));
```

Comment:
In this case, if "size" have other value than a positive integer will cause an infinite loop.
But, is just for educational purposes and it's ok, right?